webapp/src/main/resources/db/migration/V71__Add_scheduled_jobs.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE scheduled_job (
+    id CHAR(36) PRIMARY KEY NOT NULL,
+    repository_id BIGINT NOT NULL,
+    job_type SMALLINT DEFAULT NULL,
+    cron VARCHAR(50) DEFAULT NULL,
+    properties LONGTEXT NOT NULL,
+    job_status SMALLINT NOT NULL,
+    start_date TIMESTAMP NULL DEFAULT NULL,
+    end_date TIMESTAMP NULL DEFAULT NULL,
+    enabled TINYINT DEFAULT 1
+);
+
+CREATE TABLE scheduled_job_type (
+    id SMALLINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE scheduled_job_status_type (
+    id SMALLINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+ALTER TABLE scheduled_job
+    ADD UNIQUE KEY UK__REPOSITORY_ID__JOB_TYPE(repository_id, job_type);
+
+ALTER TABLE scheduled_job
+    ADD CONSTRAINT FK__SCHEDULED_JOB__IMPORT_REPOSITORY__ID
+        FOREIGN KEY (repository_id) REFERENCES repository (id);
+
+ALTER TABLE scheduled_job
+    ADD CONSTRAINT FK__JOB_TYPE__JOB_TYPE_ID FOREIGN KEY (job_type) REFERENCES scheduled_job_type(id);
+
+ALTER TABLE scheduled_job
+    ADD CONSTRAINT FK__JOB_STATUS__JOB_STATUS_ID FOREIGN KEY (job_status) REFERENCES scheduled_job_status_type(id);
+
+
+INSERT INTO scheduled_job_type(name) VALUES('THIRD_PARTY_SYNC');
+INSERT INTO scheduled_job_status_type(name) VALUES('SCHEDULED'), ('IN_PROGRESS'), ('FAILED'), ('SUCCEEDED');
+
+CREATE TABLE scheduled_job_aud (
```

Comment:
Small note to track that changes to the base table column types also get reflected the _aud version